restli-common/src/main/java/com/linkedin/restli/common/RestConstants.java

```diff
@@ -77,6 +77,7 @@
   String QUERY_BATCH_IDS_PARAM = "ids";
   String FIELDS_PARAM = "fields";
   String ALT_KEY_PARAM = "altkey";
+  String FILL_DEFAULT_VALUE_IN_RESPONSE_PARAM = "$sendDefaults";
```

Comment:
I'd suggest renaming this to be consistent with the terminology used elsewhere in this PR. `String FILL_DEFAULTS_PARAM = "$fillDefaults";` or `String FILL_IN_DEFAULTS_PARAM = "$fillInDefaults"`